Contract: lottery

2023-11-18 22:36:25,990: [96146] m.main:INFO: Registered plugins: IntrospectionAPIPlugin, <class 'manticore.ethereum.plugins.SkipRevertBasicBlocks'>, <class 'manticore.ethereum.plugins.FilterFunctions'>
2023-11-18 22:36:25,990: [96146] m.main:INFO: Beginning analysis
2023-11-18 22:36:25,995: [96146] m.e.manticore:INFO: Starting symbolic create contract
2023-11-18 22:36:28,569: [96146] m.e.manticore:INFO: Starting symbolic transaction: 0
2023-11-18 22:36:47,941: [96146] m.c.worker:ERROR: Exception in state 7: ManticoreError('Forking on unfeasible constraint set')
Traceback (most recent call last):
  File "/Users/justyna/python_environment3_7/lib/python3.7/site-packages/manticore/core/worker.py", line 137, in run
    current_state.execute()
  File "/Users/justyna/python_environment3_7/lib/python3.7/site-packages/manticore/ethereum/state.py", line 8, in execute
    return self._platform.execute()
  File "/Users/justyna/python_environment3_7/lib/python3.7/site-packages/manticore/platforms/evm.py", line 3106, in execute
    self.current_vm.execute()
  File "/Users/justyna/python_environment3_7/lib/python3.7/site-packages/manticore/platforms/evm.py", line 1322, in execute
    raise Concretize("Symbolic PC", expression=expression, setstate=setstate, policy="ALL")
manticore.core.state.Concretize

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/justyna/python_environment3_7/lib/python3.7/site-packages/manticore/core/worker.py", line 158, in run
    m._fork(current_state, exc.expression, exc.policy, exc.setstate, exc.values)
  File "/Users/justyna/python_environment3_7/lib/python3.7/site-packages/manticore/core/manticore.py", line 518, in _fork
    raise ManticoreError("Forking on unfeasible constraint set")
manticore.exceptions.ManticoreError: Forking on unfeasible constraint set
 
2023-11-18 22:36:54,093: [96146] m.c.worker:ERROR: Exception in state 3: ManticoreError('Forking on unfeasible constraint set')
Traceback (most recent call last):
  File "/Users/justyna/python_environment3_7/lib/python3.7/site-packages/manticore/core/worker.py", line 137, in run
    current_state.execute()
  File "/Users/justyna/python_environment3_7/lib/python3.7/site-packages/manticore/ethereum/state.py", line 8, in execute
    return self._platform.execute()
  File "/Users/justyna/python_environment3_7/lib/python3.7/site-packages/manticore/platforms/evm.py", line 3106, in execute
    self.current_vm.execute()
  File "/Users/justyna/python_environment3_7/lib/python3.7/site-packages/manticore/platforms/evm.py", line 1322, in execute
    raise Concretize("Symbolic PC", expression=expression, setstate=setstate, policy="ALL")
manticore.core.state.Concretize

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/justyna/python_environment3_7/lib/python3.7/site-packages/manticore/core/worker.py", line 158, in run
    m._fork(current_state, exc.expression, exc.policy, exc.setstate, exc.values)
  File "/Users/justyna/python_environment3_7/lib/python3.7/site-packages/manticore/core/manticore.py", line 518, in _fork
    raise ManticoreError("Forking on unfeasible constraint set")
manticore.exceptions.ManticoreError: Forking on unfeasible constraint set
 
2023-11-18 22:36:54,154: [96146] m.e.manticore:INFO: 0 alive states, 2 terminated states
2023-11-18 22:36:54,724: [96146] m.c.manticore:INFO: Results in /Users/justyna/mcore_2ah9lff4
2023-11-18 22:36:54,725: [96146] m.c.manticore:INFO: Total time: 26.12923812866211
